ofl/notosansarabicui/METADATA.pb

```diff
@@ -106,3 +106,4 @@ languages: "wni_Arab"  # Ndzwani Comorian
 languages: "wo_Arab"  # Wolof, Arabic
 languages: "zdj_Arab"  # Ngazidja Comorian
 languages: "zlm_Arab"  # Malay, Arabic
+primary_script: "Arab"
```

Comment:
The UI families are not used in fonts.google.com.  We probably do not need these metadata there?